plugins/module_utils/_api_data.py

```diff
@@ -1533,6 +1533,303 @@ def join_path(path):
             },
         ),
     ),
+    ('interface', 'wifi'): APIData(
+        versioned=[
+            ('7.13', '>=', VersionedAPIData(
+                fully_understood=True,
+                primary_keys=('name', ),
+                required_one_of=[['default-name', 'master-interface']],
+                fields={
+                    'aaa': KeyInfo(),
```

Comment:
The `aaa` propery is for referencing a setting under `interface wifi aaa` but one can also define the values like `aaa.called-format`, `aaa.calling-format`, `aaa.interim-update`, etc. As you set the `fully_understood` flag to `True` (which I highly encourage) would you mind adding all possible settings for `aaa.*`, `channel.*`, `configuration.*`, `datapath.*`, `interworking.*`, `security.*`, and `steering.*`?

Reply:
Also for at least 7.14 I get the following error when executing `/interface/wifi/add` without any argument:
`failure: must specify exactly one of radio-mac and master-interface`

I cannot see `default-name` on my device.
In the release notes for version 7.13 I found the following:

`*) wifi - added "radio-mac" variable for "name-format" provisioning setting;`

Reply:
OK, I've added them.

Reply:
Yep, I get the same error on 7.13. I've removed `default-name`, I don't see it so I assume it's gone.

Reply:
FYI I've added back `default-name`, it's a name-only property but I think it makes sense to keep it there as it makes configuring the interfaces easier (since you don't need to always use `radio-mac` for the masters).